discovery_utils/enrichment/crunchbase.py

```diff
@@ -425,14 +425,41 @@ def _enrich_org_investment_opportunity(organisations_enriched: pd.DataFrame) ->
     )
 
 
-def _enrich_org_smart_money(
+def _enrich_org_has_smart_money(
     organisations: pd.DataFrame,
     funding_rounds_enriched: pd.DataFrame,
 ) -> pd.DataFrame:
     """Enrich the organisations with smart money tag"""
     # find orgs that have received smart money
     smart_money_orgs = funding_rounds_enriched.query("smart_money").org_id.unique()
-    return organisations.assign(smart_money=lambda df: df.id.isin(smart_money_orgs))
+    return organisations.assign(has_smart_money=lambda df: df.id.isin(smart_money_orgs))
+
+
+def _enrich_org_is_smart_money(
+    organisations_enriched: pd.DataFrame,
+    funding_rounds_enriched: pd.DataFrame,
+    filter_mission_relevant: bool = True,
+) -> pd.DataFrame:
+    """Enrich the organisations with investor smart money tag"""
+    # Organisations relevant to one of our missions
+    if filter_mission_relevant:
+        relevant_org_ids = organisations_enriched[
+            organisations_enriched.mission_labels.fillna("").str.contains("|".join(["ASF", "AHL", "AFS"]))
```

Comment:
This would only filter down to specifically the mission label == `"ASF|AHL|AFS"`. Is this intended?
```suggestion
            organisations_enriched.mission_labels.fillna("").isin(["ASF", "AHL", "AFS"])
```

Reply:
Ahhhhhhh this is for smart money right! Ignore the above.

What about "X" or cross cutting smart money?